src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/config/HmppsIncentivesApiExceptionHandler.kt

```diff
@@ -261,14 +260,13 @@ class ListOfDataNotFoundException(dataType: String, ids: Collection<Long>) :
 class DataIntegrityException(message: String) :
   Exception(message)
 
-@JsonInclude(JsonInclude.Include.NON_NULL)
-@Schema(description = "Error Response")
+@Schema(description = "Error response")
 data class ErrorResponse(
-  @Schema(description = "Status of Error", example = "500", required = true)
+  @Schema(description = "HTTP status code", example = "500", required = true)
   val status: Int,
-  @Schema(description = "Error Code", example = "500", required = false)
+  @Schema(description = "Internal error code", example = "123", required = false)
```

Comment:
I can't think of a better description atm but is this internal if the code is used by the UI/possibly other clients 🤔? 

Maybe something like "The error code could be used to discriminate between different errors, e.g. for validation purposes" 🤷